Shell/main.cpp

```diff
@@ -181,6 +183,13 @@ int main(int argc, char** argv)
         run_rc_file(Shell::local_init_file_path);
     }
 
+    {
+        Vector<String> args;
+        for (auto* arg : script_args)
+            args.empend(arg);
+        shell->set_local_variable("ARGV", *new AST::ListValue(move(args)));
```

Comment:
This looks like a reference leak; the `*new AST::ListValue` is not adopted.

Reply:
Oh oops!

Reply:
(there are quite a few around, I think, I'll search for them and fix them along with this one)

Reply:
A `create()` helper that always returns a `NonnullRefPtr` would be nice here :)